content/v2.0/reference/cli/influx/completion/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: influx completion
+description: >
+  The 'influx completion' command outputs `influx` shell completion scripts for a
+  specified shell (`bash` or `zsh`).
+menu:
+  v2_0_ref:
+    name: influx completion
+    parent: influx
+weight: 101
+v2.0/tags: [cli, tools]
+---
+
+The `influx completion` command outputs `influx` shell completion scripts for a
+specified shell (`bash` or `zsh`).
```

Comment:
we also support `powershell`, but I haven't a clue how to set it up 😳

Reply:
I'll do a little research tomorrow morning and see what I find.